src/ObjectUtils/LinuxMap.cpp

```diff
@@ -89,28 +89,156 @@ ErrorMessageOr<std::vector<ModuleInfo>> ReadModules(int32_t pid) {
   return ParseMaps(proc_maps_data);
 }
 
+namespace {
+
+// Because of the requirements on the arguments of mmap, if the .text section of a Portable
```

Comment:
The first sentence is pretty hard to read. Can you split it up?

Reply:
I feared so. I hope it's better now.